en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -169,14 +162,6 @@ noisy, here's some pointers for auto-generated files to trigger and include in a
 - `public/_resources/`: Frontend resources copied from the (inaccessible) `vendor/` folder
   via [silverstripe/vendor-plugin](https://github.com/silverstripe/vendor-plugin).
   See [Templates: Requirements](/developer_guides/templates/requirements#exposing-resources-webroot).
-- `.graphql-generated/` and `public/_graphql/`: Schema and type definitions required by CMS and any GraphQL API endpoint.
-  Generated by
-  [silverstripe/graphql](https://github.com/silverstripe/silverstripe-graphql). See
-  [building the schema](/developer_guides/graphql/getting_started/building_the_schema) and
-  [deploying the schema](/developer_guides/graphql/getting_started/deploying_the_schema).
-- Various recipes create default files in `app/` and `public/` on `composer install`
-  and `composer update` via
-  [silverstripe/recipe-plugin](https://github.com/silverstripe/recipe-plugin).
```

Comment:
Why are we removing this bit about recipe-plugin?

Reply:
Added back in